### PR TITLE
docs: DEV-3288: Add doc how to load data from LS from ML backend

### PR DESCRIPTION
**Pull request by KonstantinKorotaev**

### CODE REVIEW

**Review by makseq — CHANGES_REQUESTED** [review]

File: docs/source/guide/export.md [hunk 1]
```diff
@@ -319,3 +319,44 @@ Review the full list of JSON properties in the [API documentation](api.html).
 
 
 <!-- md annotation_ids.md -->
+
+## Access data outside of Label Studio
+
+### Accessing data from Label Studio box
```
[the remark this answers is not in the document]
Comment (KonstantinKorotaev) [reply docs/source/guide/export.md:325]: Changed to instance

File: docs/source/guide/export.md [hunk 2]
```diff
@@ -319,3 +319,44 @@ Review the full list of JSON properties in the [API documentation](api.html).
...
+
+There are several ways of storing tasks data in Label Studio:
```
[the remark this answers is not in the document]
Comment (KonstantinKorotaev) [reply docs/source/guide/export.md:327]: Fixed

File: docs/source/guide/export.md [hunk 3]
```diff
@@ -319,3 +319,44 @@ Review the full list of JSON properties in the [API documentation](api.html).
...
+- Cloud storages 
+- External web links 
+- Uploaded files
+- Local files directory
+
+
+Label Studio stores local files in Project level structure. Each project has it's own folder for files.
+
+You can use label_studio_tools.core.utils.io.get_local_path to get task data - it will transform path or URL from task data to local path.
```
[the remark this answers is not in the document]
Comment (KonstantinKorotaev) [reply docs/source/guide/export.md]: Fixed

File: docs/source/guide/export.md [hunk 4]
```diff
@@ -319,3 +319,44 @@ Review the full list of JSON properties in the [API documentation](api.html).
...
+In case of local path it will return full local path and download resource in case of using `download_resources` parameter.
+
+Provide `Hostname` and `access_token` for accessing external resource.
+
+### Accessing data outside of Label Studio box
```
[the remark this answers is not in the document]
Comment (KonstantinKorotaev) [reply docs/source/guide/export.md]: Fixed

File: docs/source/guide/export.md [hunk 5]
```diff
@@ -319,3 +319,44 @@ Review the full list of JSON properties in the [API documentation](api.html).
...
+### Accessing data from Label Studio box
+
+There are several ways of storing tasks data in Label Studio:
+- Cloud storages 
+- External web links 
+- Uploaded files
+- Local files directory
+
+
+Label Studio stores local files in Project level structure. Each project has it's own folder for files.
```
[the remark this answers is not in the document]
Comment (KonstantinKorotaev) [reply docs/source/guide/export.md:335]: Fixed

File: docs/source/guide/export.md [hunk 6]
```diff
@@ -319,3 +319,44 @@ Review the full list of JSON properties in the [API documentation](api.html).
 
 
 <!-- md annotation_ids.md -->
+
+## Access data outside of Label Studio
```
[the remark this answers is not in the document]
Comment (KonstantinKorotaev) [reply docs/source/guide/export.md]: Fixed

**Review by makseq — APPROVED** [review]

**Review by makseq — APPROVED** [review]

**Pull request merged**